lib/src/react_client/factory_util.dart

```diff
@@ -126,11 +143,14 @@ JsMap generateJsProps(Map props,
     {bool shouldConvertEventHandlers = true,
     bool convertRefValue = true,
     bool convertCallbackRefValue = true,
+    List<String> additionalRefPropKeys = const [],
     bool wrapWithJsify = true}) {
   final propsForJs = JsBackedMap.from(props);
 
   if (shouldConvertEventHandlers) convertEventHandlers(propsForJs);
-  if (convertRefValue) convertRefValue2(propsForJs, convertCallbackRefValue: convertCallbackRefValue);
+  if (convertRefValue)
+    convertRefValue2(propsForJs,
+        convertCallbackRefValue: convertCallbackRefValue, additionalRefPropKeys: additionalRefPropKeys);
```

Comment:
#nit [prefer using curly braces for multiline if statement](https://dart.dev/guides/language/effective-dart/style#do-use-curly-braces-for-all-flow-control-structures) (probably just got wrapped during formatting)